NOTICE

```diff
@@ -1,5 +1,5 @@
 OpenSearch
```

Comment:
Can you update this line to ` OpenSearch (https://opensearch.org/)`

Reply:
Will do!